cl/settings/05-private.example

```diff
@@ -9,13 +9,11 @@ DEBUG = True
 DEVELOPMENT = True
 
 if DEVELOPMENT:
-    ALLOWED_HOSTS = ['*']
+    ALLOWED_HOSTS = ["*"]
```

Comment:
Can we get this file out of this PR? I don't want to mess up our git blame results permanently.